### PR TITLE
Replace `nuxt.showOpen` with `nuxt.showReady`

### PR DESCRIPTION
**Pull request by mubaidr**
`nuxt.showOpen` is not a function

### CODE REVIEW

**Comment by mubaidr** [comment]
Removed `nuxt.showReady` and replace process.exit with throw.

**Comment by mubaidr** [comment]
You are right, we don't need to catch error at all. This: 
```bash
(node:20732) [DEP0018] DeprecationWarning: Unhandled promise rejections are depr
ecated. In the future, promise rejections that are not handled will terminate th
e Node.js process with a non-zero exit code.
```

after all we are just logging and exiting process.

File: ja/api/configuration-dev.md [hunk 1]
```diff
@@ -42,15 +42,11 @@ app.use(nuxt.render)
...
-  .catch((error) => {
-    console.error(error)
-    process.exit(1)
-  })
 }
 
 // サーバーを Listen する
 app.listen(port, '0.0.0.0').then(() => {
-  nuxt.showOpen()
+  console.log(`Server listning on port: ${port}`)
```
Comment (manniL) [inline ja/api/configuration-dev.md]: Typo here
```suggestion
  console.log(`Server listening on port: ${port}`)
```

Comment (mubaidr) [reply ja/api/configuration-dev.md]: 🤦‍♂️

File: de/api/configuration-dev.md [hunk 2]
```diff
@@ -23,31 +23,31 @@ Example:
...
+};
 ```
 
 `server.js`
 
 ```js
-const { Nuxt, Builder } = require('nuxt')
-const app = require('express')()
-const port = process.env.PORT || 3000
+const { Nuxt, Builder } = require('nuxt');
```
Comment (manniL) [inline de/api/configuration-dev.md]: We are coming closer! However, please use semicolon-less style and only include necessary changes :relaxed:

**Comment by mubaidr** [comment]
@manniL please https://github.com/nuxt/docs/pull/1065

**Pull request closed without merging**